language-support/hs/bindings/src/DA/Ledger/Types.hs

```diff
@@ -227,8 +228,7 @@ data Timestamp
         seconds :: Integer, -- TODO: Int64?
         nanos   :: Integer }  deriving (Eq,Ord,Show)
 
-data Status = Status-- TODO: from standard google proto, determining success/failure
- deriving (Eq,Ord,Show)
+type Status = LL.Status
```

Comment:
I’m not sure that type synonym adds anything over using `LL.Status` directly.

Reply:
I agree. Just re-exporting `LL.Status` should be good enough.

Reply:
ok, will do